tests/unit/constraints/test_base.py

```diff
@@ -284,7 +284,7 @@ def test_transform_invalid_table_data(self):
         # Run
         instance = Constraint(handling_strategy='transform')
         instance._transform = lambda x: x
-        instance._constraint_columns = ('a')
+        instance.constraint_columns = ('a')
```

Comment:
I think that this should be `('a', )`, otherwise it is equivalent to `'a'`, without the tuple.